packages/data-stores/src/reader/hooks/index.ts

```diff
@@ -1,5 +1,7 @@
 import { useSelect } from '@wordpress/data';
+import { useMemo } from '@wordpress/element';
```

Comment:
Does that make any difference that we import useMemo from `@wordpress/element` as opposed to `react`?